examples/misc/legends/poly-legend-ma.html

```diff
@@ -74,8 +74,8 @@ <h1>Percent Master's Degree</h1>
 
       colorLegend.data.forEach((legend) => {
         const colorHex = rgbToHex(legend.value);
-        const keyMin = legend.key[0].toFixed(2).replace('Infinity', '<');
-        const keyMax = legend.key[1].toFixed(2).replace('Infinity', '>');
+        const keyMin = legend.key[0].toFixed(2).replace('Infinity', '∞');
```

Comment:
I don't understand the reason for this change. I thought @makella actually complained about showing `Infinity`, but the original code replaced it to be more user friendly. :confused:

Reply:
Sorry, I didn't mean to be rude, my English fault, sorry.

I answered to the related issue too https://github.com/CartoDB/carto-vl/issues/1110#issuecomment-436231428

Reply:
i couldn't get rid of the `-` so i reverted back to the infinity sign